src/widgets/FormControl.js

```diff
@@ -271,9 +272,11 @@ const mergeProps = (stateProps, dispatchProps, ownProps) => {
     cancelButtonText,
     onCancel,
     canSave = true, // if not specified by calling component, set default
+    onUpdate,
   } = ownProps;
   const onInitialiseForm = () => initialiseForm(inputConfig);
-  const onUpdateForm = (key, value) => !isDisabled && updateForm(key, value);
+  const onUpdateForm = (key, value) =>
```

Comment:
I would've thought it doesn't matter what this fn is if the form is disabled, but it does have me curious if there was a reason it was not calling `updateForm` when it was disabled.

Currently if there is an `onUpdate` passed and it is disabled then `updateForm` is set- probably nothing 🤔

Reply:
oopsie. 6634c014